src/js/angular/ttyg/templates/chat-panel.html

```diff
@@ -1,14 +1,10 @@
 <link href="css/ttyg/chat-panel.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
 
-<div class="chat-panel h-100">
+<div class="chat-panel h-100" ng-class="chat.chatHistory.isEmpty() ? 'no-content' : ''">
```

Comment:
The ng-class condition has to be `ng-class="chat.chatHistory.isEmpty() && !askingChatItem ? 'no-content' : ''` to display the first question and loader immediately after clicking the ask button.